pom.xml

```diff
@@ -210,6 +214,7 @@
                 <groupId>com.github.cerveada</groupId>
                 <artifactId>scalatest-maven-plugin</artifactId>
                 <configuration>
+                    <stdout>I</stdout>
```

Comment:
Only extra add from me here on this PR - this option re-prints the failed tests w/ a tiny bit of log at the end of a run. Found it helpful while trying to figure out what needed to be fixed. Happy to remove if desired.